plugins/inputs/nvidia_smi/README.md

```diff
@@ -0,0 +1,25 @@
+# `nvidia-smi` Input Plugin
+
+This plugin uses a query on the [`nvidia-smi`](https://developer.nvidia.com/nvidia-system-management-interface) binary to pull GPU stats including memory and GPU usage, temp and other.
+
+```
+nvidia-smi --query-gpu=fan.speed,memory.total,memory.used,memory.free,pstate,temperature.gpu,name,uuid,compute_mode,utilization.gpu,utilization.memory,index --id={{ gpu_index }} --format=csv,noheader,nounits
+```
+
+#### Tags
+
+- `name`: The type of GPU e.g. `GeForce GTX 170 Ti`
+- `compute_mode`: The compute mode of the GPU e.g. `Default`
+- `index`: The port index where the GPU is connected to the motherboard e.g. `1`
+- `pstate`: Overclocking state for the GPU e.g. `P0`
+- `uuid`: A unique identifier for the GPU e.g. `GPU-f9ba66fc-a7f5-94c5-da19-019ef2f9c665`
```

Comment:
Do you know when this changes?  Can you check on reboot and, if possible, on change of slot?

Reply:
Good question! On reboot it does not change. I'll have to try change of slot next time I have access to the machines.